ui/decredmaterial/collapsible.go

```diff
@@ -6,280 +6,78 @@ import (
 )
 
-type MoreItem struct {
-	Text   string
-	Button *widget.Clickable
-	label  Label
-}
-
 type Collapsible struct {
 	IsExpanded      bool
 	Button          *widget.Clickable
-	BackgroundColor color.RGBA
-	expandedIcon    *widget.Icon
-	collapsedIcon   *widget.Icon
+	BackgroundColor color.NRGBA
+	expandedIcon    *widget.Image
+	collapsedIcon   *widget.Image
 	card            Card
 }
 
-type CollapsibleWithOption struct {
-	Items             []MoreItem
-	Collapsible       *Collapsible
-	MoreIcon          IconButton
-	isOpened          bool
-	color             color.RGBA
-	selectedMoreIndex int
-	theme             *Theme
-}
-
 func (t *Theme) Collapsible() *Collapsible {
 	c := &Collapsible{
 		BackgroundColor: t.Color.Surface,
 		Button:          new(widget.Clickable),
-		expandedIcon:    t.chevronUpIcon,
-		collapsedIcon:   t.chevronDownIcon,
 		card:            t.Card(),
+		collapsedIcon:   t.collapseIcon,
+		expandedIcon:    t.expandIcon,
 	}
 	c.card.Color = c.BackgroundColor
+	c.collapsedIcon.Scale, c.expandedIcon.Scale = 1, 1
 
 	return c
 }
 
-func (t *Theme) CollapsibleWithOption(Items []MoreItem) *CollapsibleWithOption {
-	c := &CollapsibleWithOption{
-		Items: make([]MoreItem, len(Items)+1),
-		color: t.Color.Background,
-		theme: t,
-		Collapsible: &Collapsible{
-			BackgroundColor: t.Color.Surface,
-			Button:          new(widget.Clickable),
-		},
-		MoreIcon: IconButton{
-			material.IconButtonStyle{
-				Icon:       t.navMoreIcon,
-				Size:       unit.Dp(25),
-				Background: color.RGBA{},
-				Color:      t.Color.Text,
-				Inset:      layout.UniformInset(unit.Dp(0)),
-				Button:     new(widget.Clickable),
-			},
-		},
-	}
-
-	for i := range Items {
-		Items[i].Button = new(widget.Clickable)
-		Items[i].label = c.theme.Body1(Items[i].Text)
-		c.Items[i+1] = Items[i]
-	}
-
-	return c
-}
-
-func (c *Collapsible) Layout(gtx layout.Context, header func(C) D, content func(C) D) layout.Dimensions {
+func (c *Collapsible) getHeaderParts(header, option func(C) D) []layout.FlexChild {
 	icon := c.collapsedIcon
 	if c.IsExpanded {
 		icon = c.expandedIcon
 	}
 
-	for c.Button.Clicked() {
-		c.IsExpanded = !c.IsExpanded
-	}
-
-	return c.card.Layout(gtx, func(gtx C) D {
-		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return layout.Stack{}.Layout(gtx,
-					layout.Stacked(func(gtx C) D {
-						gtx.Constraints.Min.X = gtx.Constraints.Max.X
-						return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								return header(gtx)
-							}),
-							layout.Rigid(func(gtx C) D {
-								return layout.Inset{
-									Right: unit.Dp(10),
-								}.Layout(gtx, func(C) D {
-									return icon.Layout(gtx, unit.Dp(20))
-								})
-							}),
-						)
-					}),
-					layout.Expanded(c.Button.Layout),
-				)
-			}),
-			layout.Rigid(func(gtx C) D {
-				if c.IsExpanded {
-					return content(gtx)
-				}
-				return layout.Dimensions{}
```

Comment:
Collapsible widget used in the wallets page and other parts of the app have different layouts. This should be restored